docs/ref/contrib/admin/index.txt

```diff
@@ -2507,11 +2507,32 @@ Templates can override or extend base admin templates as described in
     * ``site_title``: :attr:`AdminSite.site_title`
     * ``site_url``: :attr:`AdminSite.site_url`
     * ``has_permission``: :meth:`AdminSite.has_permission`
+    * ``available_apps``: a list of applications from the
+      :doc:`application registry </ref/applications/>` available for the
```

Comment:
you can probably move part of this to the previous line and break after "registry"